app/api_parser.py

```diff
@@ -0,0 +1,38 @@
+import json
+import requests
+from config import FORECAST_API_KEY, GOOGLE_API_KEY
+
+
+class ForeCast:
+
+    def __init__(self, latitude, longitude):
+        self.url = 'https://api.forecast.io/forecast/{}/{},{}'.format(FORECAST_API_KEY, latitude, longitude)
+
+    def get_request(self):
+        return json.loads((requests.get(self.url)).text)
+
+
+class GeoCode:
+
+    def __init__(self, address):
+        self.url = 'https://maps.googleapis.com/maps/api/geocode/json?address={}&key={}'.format(address, GOOGLE_API_KEY)
```

Comment:
I made a query for “Akazienstraße 1”:

WEATHER FORECAST FOR AKAZIENSTRASSE 1, 06366 KÖTHEN (ANHALT), GERMANY: Place Type: street_number Temperature: 4.56°C Humidity: 0.93 Weather Summary: Light Rain

How's it?